src/components/ConfirmationPopUp.tsx

```diff
@@ -26,73 +22,41 @@ const actionModalStyle = StyleSheet.create({
   }
 });
 
-export default class ConfirmationPopUp extends PureComponent<
-  ConfirmationPopUpProps,
-  ConfirmationPopUpState
-> {
-  state = {
-    isOpen: false
-  };
-
-  private toggle = () =>
-    this.setState({
-      isOpen: !this.state.isOpen
-    });
-
-  render():
-    | React.ReactElement<any>
-    | string
-    | number
-    | {}
-    | React.ReactNodeArray
-    | React.ReactPortal
-    | boolean
-    | null
-    | undefined {
-    const {
-      disabled,
-      children,
-      title,
-      description,
-      onConfirmPress,
-      onRejectPress,
-      confirmButtonText,
-      rejectButtonText
-    } = this.props;
-    const { isOpen } = this.state;
-    return (
+export default function ConfirmationPopUp({
+  title,
+  description,
+  onConfirmPress,
+  onRejectPress,
+  confirmButtonText,
+  rejectButtonText,
+  visible,
+  onClose
+}: ConfirmationPopUpProps) {
+  return (
+    <ActionModal
```

Comment:
Is it a good idea to do
`<ActionModal {... ConfirmationPopUpProps}/>` You'll be sending in some extra ones, which could be a problem if the other ones change. Not sure.